Source/WebCore/html/HTMLMediaElement.h

```diff
@@ -336,7 +336,7 @@ class HTMLMediaElement
 
 #if ENABLE(MEDIA_SOURCE)
 //  Media Source.
-    void detachMediaSource();
+    void detachMediaSource(bool shuttingDown = false);
```

Comment:
Boolean parameters like this are frowned upon in WebKit because they lead to poor readability at call sites. This should use an enum class.

Reply:
maybe we should add it to coding guidelines?

Reply:
https://webkit.org/code-style-guidelines/#names-enum-to-bool

Reply:
is it okay to have default parameter value still ?

Reply:
> is it okay to have default parameter value still ?

I think so.